src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/dbscripts/sql_logs.sql

```diff
@@ -14,7 +14,13 @@
 
 -- This SQL extracts all non-truncated SQL statements from DBC.QryLogSQLV by default
 SELECT
+  "ProcID",
+  "CollectTimeStamp" AT TIME ZONE INTERVAL '0:00' HOUR TO MINUTE AS "CollectTimeStamp",
   "{{#if vars.columnNameRowNumber}}{{vars.columnNameRowNumber}}{{else}}SqlRowNo{{/if}}" AS "SqlRowNo",
   "{{#if vars.columnNameQueryID}}{{vars.columnNameQueryID}}{{else}}QueryID{{/if}}" AS "QueryID",
-  "{{#if vars.columnNameQueryText}}{{vars.columnNameQueryText}}{{else}}SqlTextInfo{{/if}}" AS "QueryText"
-FROM "{{baseDatabase}}"."{{#if vars.tableName}}{{vars.tableName}}{{else}}QryLogSQLV{{/if}}"
\ No newline at end of file
+  "{{#if vars.columnNameQueryText}}{{vars.columnNameQueryText}}{{else}}SqlTextInfo{{/if}}" AS "SqlText"
```

Comment:
Why do we actually have  vars for column names in this script? Other scripts don't use them? Is there any difference between Teradata versions?

Reply:
If we can ask consumers to always use QryLogSQLV – these variables aren't required and can be safely removed.
I was going to discuss it with @haselbach today

Reply:
Great. I just wanted to point out that generally I would avoid creating to complex templates and would go towards optional queries instead of multi-shape queries.

Reply:
I have the same opinion – for a long term, having optional scripts is a way to go. Christian although would like to keep it for now.